packages/common/src/project/readers/reader.ts

```diff
@@ -42,10 +39,10 @@ export class ReaderFactory {
       return new IPFSReader(locationWithoutSchema, options.ipfs);
```

Comment:
logic can be improved here. if `ipfs://` is provided but then is tested to be not valid, shouldn't we throw error?